README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
@hferentschik I manually tested it and its working now. Previously, it wasn't.

I have some idea about the test using rspec which we discussed for `install-cli` proxy feature.

I idea is to run HTTPServer as forward proxy server to a dummy simple http server. Then, pass details and check whether it passes through forward proxy server or not. Similar to what we discussed.

Note, that I am using a dummy simple http server as I couldn't find a way for proxy server to always return some fixed content.